README.md

```diff
@@ -20,49 +41,46 @@ Docusaurus provides a feature for managing versions. It uses [Fallback functiona
 
 ### How does this tool work?
 
-This tool was created to solve the problems mentioned above. It provides a new command, `docusaurus-snapshot-version`, for creating new version. It's a wrapper for the built-in command Docusaurus uses to create new versions.
+This tool was created to solve the problems mentioned above. It provides a new package, `docusaurus-snapshot-version`, for creating new versions. It's a wrapper for the built-in command Docusaurus uses to create new versions.
 
 The command uses the following conventions for managing static assets:
 
+#### Docs-Specific Static Assets
+
 - All static assets for the "next release" are at the root of the `docs/assets/` directory as a flat structure, that is, there are no sub-directories.
 - All static assets for "versioned docs" are at the root of the directory `docs/assets/version-<version-number>/` in a flat structure, that is, there are no sub-directories.
 
-    Ideally, the versioned assets would reside with the versioned documents (i.e. `website/versioned_docs/version-<version-number>/assets/`), but this cannot be achieved without changing the core logic of Docusaurus V1.
+  Ideally, the versioned assets would reside with the versioned documents (i.e. `website/versioned_docs/version-<version-number>/assets/`), but this cannot be achieved without changing the core logic of Docusaurus V1.
+
+#### Site-Specific Static Assets
+
+Site-specific static asset files are in the `website/static/` directory, and static directories at this location can be versioned.
+
+- If a static directory is versioned for the first time, everything in the static directory is copied into the `<version-number>` and the `next` (for next release) directories. These new directories are under the same static directory.
+- If a static directory has been versioned before, everything in the `next` directory is copied into the `<new-version-number>` directory.
+
+e.g. Once `website/static/img` directory is versioned, files from that directory are in the following directories:
+- `website/static/img/next`
+- `website/static/img/<version-number>`
```

Comment:
Re-reading this, should we perhaps use a different folder name as an example. In our case, we are interested in versioning "javadocs" being stored there. Maybe that's a more practical example because normally docs images would be used in the "Docs-Specific static assets". What do you think?

Reply:
Yeah let's do that because Javadocs is the reason why we had to implement these new features anyway.